server/bin/gold/test-7.0.1.txt

```diff
@@ -2335,7 +2335,7 @@ Template: pbench-unittests.v5.result-data
                 "type": "date"
             },
             "@timestamp_original": {
-                "type": "text"
+                "type": "keyword"
```

Comment:
Did we want to add the `"index": "false"` here?

Reply:
I don't know if there's any significant benefit, and I'd have to make another testing round to be sure it doesn't break anything; but we could do that if someone feels strongly about it.

Reply:
It would be good to remove the indexes for this since this field will be in many documents.

Reply:
Likely true of other fields as well; we really should do a full review. I'll plan on setting something up soon after I get this pass stabilized.

But, for now, sure; I'll try marking '@timestamp_original' as "index": false here and in `result-data.json` as well.